package.json

```diff
@@ -102,7 +106,7 @@
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "tslint -p test -t stylish",
     "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
-    "test": "jest",
+    "test": "nyc --extension .ts mocha --forbid-only \"test/**/che.test.ts\"",
```

Comment:
seems jest (my previous PR) is removed ?
jest is not working ?

Reply:
Yes I noticed that. Will fix it.